Automattic/wp-calypso
# Block context switch when uploading image

#### Changes proposed in this Pull Request

* When uploading images the user can context switch while an async image upload is happening leading to a dirty unpredictable state. This change aims to make image upload a blocking action for the user.



#### Testing instructions
* Go through the DIFM flow, checkout and reach the website-content state
* Upload an image anywhere and make sure switching across form sections is blocked
* Clicking on other images inside sections should be allowed
